File: 557-reverse-words-in-a-string-iii/557-reverse-words-in-a-string-iii.java
class Solution {
    public String reverseWords(String s) {
        String[] arr = s.split(" ");
        StringBuilder str = new StringBuilder();
        
        for(String i:arr){
            str.append(new StringBuffer(i).reverse()+" ");
        }
        
        

    return str.toString().trim();        
    }    
}

File: 557-reverse-words-in-a-string-iii/557-reverse-words-in-a-string-iii.js
/**
 * @param {string} s
 * @return {string}
 */
const reverseWords = function(s) {
    let ans = s.split(" ");
   
    for(let i=0;i<ans.length;i++){
        
        let arr = ans[i].split("");
        
        arr.reverse();
        let joint = arr.join("");
        arr.pop();
        ans[i] = joint;

        
    }
    
    return ans.join(" ");
};